manim/scene/graph_scene.py

```diff
@@ -654,36 +646,35 @@ def get_riemann_rectangles(
     def get_riemann_rectangles_list(
         self, graph, n_iterations, max_dx=0.5, power_base=2, stroke_width=1, **kwargs
     ):
-        """
-        This method returns a list of multiple VGroups of Riemann
-        Rectangles. The initial VGroups are relatively inaccurate,
+        """This method returns a list of multiple :class:`~.VGroup`s of Riemann
+        Rectangles. The initial :class:`~.VGroup`s are relatively inaccurate,
```

Comment:
This has a very long one-line summary. Also, rendering the reference seems not to work here (but I'm not sure why, I was under the impression that a space after the backtick is not necessary).

```suggestion
        """Returns a list of Riemann Rectangles.
        
        This method returns a list of multiple :class:`~.VGroup`s of Riemann
        Rectangles. The initial rectangles are relatively inaccurate,
```